crypto/decrepit/cfb/cfb.c

```diff
@@ -51,30 +104,80 @@ static int aes_cfb128_cipher(EVP_CIPHER_CTX *ctx, uint8_t *out,
   return 1;
 }
 
+static const EVP_CIPHER aes_128_cfb1 = {
+    NID_aes_128_cfb1,  1 /* block_size */,  16 /* key_size */,
```

Comment:
Should use `//` for comments.

Reply:
This is not required. Other EVP define also uses /* */

Reply:
meh, I'm just following our normal convention. Because it is wrong one place, does not mean we can't follow it another place :)

Reply:
> I'm just following our normal convention

When adding this style comments, below conventions were considered. 
* All EVP PKEY methods(including the existing CFB one) use `/* */` comments style instead of `//`. * I saw upstream uses `convert_comments.go` in some directories but not applied in the pmethod files yet.
* Inline argument comment usage shows different comment styles are needed.
https://github.com/awslabs/aws-lc/blame/b3ed071ecc4efb77afd0a025ea1078da19578bfd/ssl/ssl_lib.cc#L1998

> Because it is wrong one place

I don't think it's wrong usage, and not just one place.